features/currency/managing_exchange_rates/browsing_exchange_rates.feature

```diff
@@ -0,0 +1,18 @@
+@managing_exchange_rates
+Feature: Browsing exchange rates
+    In order to manage exchange rates used in the store
+    As an Administrator
+    I want to browse exchange rates
+
+    Background:
+        Given the store has currency "Euro", "British Pound"
+        And the store has currency "Bhutanese Ngultrum"
```

Comment:
Given the store has currency "Euro", "British Pound" **and**  "Bhutanese Ngultrum"